src/ValueFormatters/QuantityFormatter.php

```diff
@@ -148,6 +151,22 @@ protected function formatQuantityValue( QuantityValue $quantity ) {
 	}
 
 	/**
+	 * @param UnboundedQuantityValue $quantity
+	 *
+	 * @return string
+	 */
+	protected function formatUnboundedQuantityValue( UnboundedQuantityValue $quantity ) {
+		$amount = $quantity->getAmount();
+		$roundingExponent = $this->options->getOption( self::OPT_APPLY_ROUNDING );
+
+		if ( !is_bool( $roundingExponent ) ) {
+			$amount = $this->decimalMath->roundToExponent( $amount, (int)$roundingExponent );
+		}
```

Comment:
But if all you have is the string "1.499999999999999", no bounds and an option set to `true`, how do you round?